<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tablica Kanban</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addTaskButton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj zadanie</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollAreaToDo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>311</width>
      <height>551</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="widgetsToDo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>309</width>
       <height>549</height>
      </rect>
     </property>
     <property name="columnNumber" stdset="0">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollAreaDoing">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>311</width>
      <height>551</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="widgetsDoing">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>309</width>
       <height>549</height>
      </rect>
     </property>
     <property name="columnNumber" stdset="0">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollAreaDone">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>311</width>
      <height>551</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="widgetsDone">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>309</width>
       <height>549</height>
      </rect>
     </property>
     <property name="columnNumber" stdset="0">
      <number>2</number>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollAreaNotes">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>110</y>
      <width>311</width>
      <height>551</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="widgetsNotes">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>309</width>
       <height>549</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addNoteButton">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj notatkÄ™</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelToDo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>To Do</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDoing">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Doing</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDone">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelNotes">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>60</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="editToDo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
    </property>
    <property name="columnNumber" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="editDoing">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
    </property>
    <property name="columnNumber" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="editDone">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>70</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
    </property>
    <property name="columnNumber" stdset="0">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="editNotes">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>70</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
    </property>
    <property name="columnNumber" stdset="0">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Zalogowano jako:&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; User&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logOutButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Wyloguj</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SystemLogOut"/>
    </property>
   </widget>
   <widget class="QPushButton" name="editUserButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Edytuj</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1320</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
